functions/v2/log/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_cloudevent_log_stackdriver]
+import base64
+import json
+
+# [START functions_cloudevent_log_helloworld]
+import functions_framework
+
+# [END functions_cloudevent_log_stackdriver]
+
+
+@functions_framework.cloud_event
+def hello_world(event):
+    """Cloud Event Function.
+    Args:
+        event: Cloud event for the function trigger
+    """
+    print("Hello, stdout!")
+
+
+# [END functions_cloudevent_log_helloworld]
+
+
+# [START functions_cloudevent_log_stackdriver]
+@functions_framework.cloud_event
+def process_log_entry(event):
```

Comment:
I'm unclear on the context here. A Cloud Function has only one entry point specified, so why does this code seem to have two? Would it be deployed twice, once with each entry point? Should the two entry points be two separate samples?

Reply:
I was updating https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/functions/log/main.py that had both samples together. I will separate them into different samples which is cleaner.

Reply:
I split the sample into two distinct ones in their respective directories.